pkg/plugins/driver.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto3";
+
+package plugins;
+
+option go_package = "github.com/lima-vm/lima/pkg/plugins";
+
+service VMDriver {
```

Comment:
Eventually this service should also have a method "Fulfill the empty fields such as mountType"